lib/mochaReporter.js

```diff
@@ -147,16 +156,20 @@ class RPReporter extends Mocha.reporters.Base {
   finishTest(test, status) {
     try {
       if (this.currentTest) {
-        this.rpClient.finishTestItem(this.currentTest.tempId, {
+        let launchFinishObj = {
```

Comment:
Is it a `launchFinishObj`? `testItemFinishObj` will be less confusing

Reply:
Fixed